packages/protocol/test-sol/unit/common/Registry.t.sol

```diff
@@ -18,20 +19,15 @@ contract RegistryTest is Test {
   address owner;
 
   function setUp() public {
+    super.setUp();
+    whenL2WithEpochManagerInitialization();
```

Comment:
does it make sense to keep L2 in function names anymore ?

Reply:
For now, I would keep, because we still have the e2e test that get started on L1 then migrate to L2. this is mainly because of how L2 is initialized using the `EpochManagerEnabler`